web/libs/editor/src/components/Comments/Comment/CommentItem.scss

```diff
@@ -115,6 +115,15 @@
     overflow: hidden;
   }
 
+  .comment_form {
+    // to hide the weird glow below the focused form, because it has outline but it's already cut off
+    overflow: hidden;
```

Comment:
I mentioned this in the other PR, this should be looked at from the FE Guild componentization effort perspective. As the various states are important to visualize correctly, but the reasons they cannot be handled here is likely outside of the scope of any general ticket.